src/main/java/appeng/core/api/definitions/ApiBlocks.java

```diff
@@ -497,10 +493,8 @@ public void customize( IBlockRendering rendering, IItemRendering itemRendering )
 				.rendering( new CableBusRendering( partModels ) )
 				// (handled in BlockCableBus.java and its setupTile())
 				// .tileEntity( TileCableBus.class )
-				.postInit( ( block, item ) ->
-				{
-					( (BlockCableBus) block ).setupTile();
-				} )
+				// TODO: why the custom registration?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)